### PR TITLE
RATEST-RATEST-275b:NEW E2E Test: Manage OCL Subscription Workflow

### PR DESCRIPTION
**Pull request by jwnasambu**
**Description of what I worked on**

I have created the Open Concept Lab page, feature file and step definition file in preparation to automated testing.

**Issue worked on**:
https://issues.openmrs.org/browse/RATEST-275

### CODE REVIEW

**Comment by sherrif10** [comment]
Would have tried re run this workflow but again, @jwnasambu I can see alot of errors concerning about some pages not getting called, I would think we can first run this tests locally to confirm first

**Comment by sherrif10** [comment]
Am available today we sync and finish this please if you don't mind.thnks

File: src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    }
+    
+    @Then("the system loads Open Concept Lab page")
+    public void systemLoadsOpenConceptPage() {
+        assertPage(openconceptlabpage.waitForPage());
+    }
+    
+   @When ("a user clicks on Setup subscription button")
+    public void loadSubscriptionPage() {
+      subscriptionPage = openconceptlabpage.clickOnsetupSubscriptionButton();
```
Comment (kdaud) [inline src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java:75]: @jwnasambu have you taken a look at the logs at? https://github.com/openmrs/openmrs-contrib-qaframework/runs/4437237295?check_suite_focus=true#step:7:11159

is clickOnSetupSubscriptionButton() existing any where within [OpenConceptLabPage](https://github.com/openmrs/openmrs-contrib-qaframework/blob/37bbc55d70c17b18051dedbc2d816a7364564154/src/test/java/org/openmrs/contrib/qaframework/page/OpenConceptLabPage.java#L1-L82)?

Comment (jwnasambu) [reply src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java:75]: @sharif thanks for the review actually as @kdaud pointed out the error was originating from a method  which doest exist.

File: src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+import org.openmrs.contrib.qaframework.page.SubscriptionPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+ 
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
```
Comment (kdaud) [inline src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java:30]: @jwnasambu take a look at the import organisation in one of the existing file [here](https://github.com/openmrs/openmrs-contrib-qaframework/blob/da4bbde0eebcf62fcc0918efc84cd97356bc88d9/src/test/java/org/openmrs/contrib/qaframework/automation/EditDemographicsAndContactInfoSteps.java#L12-L25) and align to the convention.

**Comment by kdaud** [comment]
@jwnasambu did you look at the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/257#discussion_r763661058)?

**Comment by kdaud** [comment]
you may need to check the attached ticket url in the description section.

**Pull request closed without merging**